docker/dev/find_fastest_go_proxy.sh

```diff
@@ -0,0 +1,46 @@
+#!/bin/bash
+# Copyright 2020 The SQLFlow Authors. All rights reserved.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Define a list of mirrors without using Bash arrays.
+# c.f. https://stackoverflow.com/a/23930212/724872
+DEFAULT_URL="https://pkg.go.dev/"
+read -r -d '' GO_PROXIES <<EOM
+${DEFAULT_URL}
```

Comment:
How comes this var?

Reply:
Just use it to test if we have to add a proxy. If we can access this site fast, we do not need a proxy.

Reply:
If it is only for testing, how about remove it from this change?

If it has more use than testing, it needs a precise name.  DEFAULT_URL is too general a name.

Reply:
BTW, how should we decide if we need a Go proxy here?

Reply:
I mean we compare this site's access speed to other real `porxies`, if it actually is faster than proxies like `goproxy.cn`, we will not use any proxy.
Maybe I need to describe it in the script.

Reply:
I have never used a Go proxy here in America. I guess we don't need it here for public repo development.  A Go proxy, in my mind, is good for accelerating download in China specifically, and for redirecting employees to a corp-mirror of Go packages.